mmdet/core/evaluation/eval_hooks.py

```diff
@@ -114,15 +115,15 @@ def _do_evaluate(self, runner):
             tmpdir = osp.join(runner.work_dir, '.eval_hook')
 
         from mmdet.apis import multi_gpu_test
-        results = multi_gpu_test(
+        self.results = multi_gpu_test(
```

Comment:
It may be necessary to confirm whether the memory will increase after changing to the self attribute. Because result will be resident in memory

Reply:
I checked the current implementation and without using the same (and removed self from self.results) - there is no change in the memory requirement while training is going on. I didn't notice any increase in the memory because of the modification to eval_hooks.py. :)